ansible-collection-microsoft-sql/ansible-collection-microsoft-sql.spec

```diff
@@ -54,159 +35,49 @@ BuildRequires: ansible >= 2.9.10
 %endif
 
 %if %{without ansible}
-# Empty command. We don't have ansible-galaxy.
-%define ansible_collection_build() tar -cf %{_tmppath}/%{collection_namespace}-%{collection_name}-%{version}.tar.gz .
-%else
-%define ansible_collection_build() ansible-galaxy collection build
-%endif
-
-%if %{without ansible}
+# We don't have ansible-galaxy.
 # Simply copy everything instead of galaxy-installing the built artifact.
-%define ansible_collection_install() mkdir -p %{buildroot}%{ansible_collection_files}/%{collection_name}; (cd %{buildroot}%{ansible_collection_files}/%{collection_name}; tar -xf %{_tmppath}/%{collection_namespace}-%{collection_name}-%{version}.tar.gz)
+%define ansible_collection_build_install() tar -cf %{_tmppath}/%{collection_namespace}-%{collection_name}-%{version}.tar.gz .; mkdir -p %{buildroot}%{ansible_collection_files}/%{collection_name}; (cd %{buildroot}%{ansible_collection_files}/%{collection_name}; tar -xf %{_tmppath}/%{collection_namespace}-%{collection_name}-%{version}.tar.gz)
 %else
-%define ansible_collection_install() ansible-galaxy collection install -n -p %{buildroot}%{_datadir}/ansible/collections %{collection_namespace}-%{collection_name}-%{version}.tar.gz
+%define ansible_collection_build_install() ansible-galaxy collection build; ansible-galaxy collection install -n -p %{buildroot}%{_datadir}/ansible/collections %{collection_namespace}-%{collection_name}-%{version}.tar.gz
 %endif
 
-# For each role, call either defcommit() or deftag(). The other macros
-# (%%id and %%shortid) can be then used in the same way in both cases.
-# This way  the rest of the spec file des not need to know whether we are
-# dealing with a tag or a commit.
+# For each role, call defcommit() and the point to it with SourceN: %{archiveurlN}.
 %global archiveext tar.gz
-# list of role names
+# list of source role names
 %global rolenames %nil
 # list of assignments that can be used to populate a bash associative array variable
 %global rolestodir %nil
+# list of target rolenames to copy the roles to
+%global target_rolenames %nil
+# list of collection rolenames to convert the roles to
+%global collection_rolenames %nil
+
 %define getarchivedir() %(p=%{basename:%{S:%{1}}}; echo ${p%%.%{archiveext}})
 
+%global parenturl https://github.com/linux-system-roles/
```

Comment:
I think it is more natural to have no slash `/` at the end of the url string here, and instead where you use it, you use `/`

Reply:
and yes, I realize that linux-system-roles has the `/` at the end of the url

Reply:
I've applied this improvement here and opened a PR for linux-system-roles too, please take a look if interested:
https://src.fedoraproject.org/fork/spetros/rpms/linux-system-roles/diff/rawhide..naturalize-url

Reply:
Correct URL to the PR: https://src.fedoraproject.org/rpms/linux-system-roles/pull-request/14